PyInstaller/compat.py

```diff
@@ -437,3 +437,11 @@ def importlib_load_source(name, pathname):                # Import module from a
         mod_loader = importlib.machinery.SourceFileLoader(name, pathname)
         return mod_loader.load_module()
 
+# String types to replace `isinstance(foo, str)`
+# Use `isinstance(foo, string_types)` instead.
+
+if is_py2:
```

Comment:
Ah, the first time I simply wrote `import six` and then `isinstance(foo, six.string_types)` but then decided that if this is the only thing from `six` I'm using, I may as well paste it into compat.py.  But you're right, six uses `basestring`. 

I did a quick search for `isinstance(.*, str)` and there were a couple more uses in macholib. I didn't want to change them since I don't have OS X to test on. There are a couple more uses in modulegraph too, but since it is checking module names (which aren't allowed to be unicode on py2 anyway) it shouldn't matter.